packages/react-search-ui/src/containers/__tests__/Facet.test.js

```diff
@@ -43,14 +43,12 @@ const params = {
   },
   addFilter: jest.fn(),
   removeFilter: jest.fn(),
-  setFilter: jest.fn()
+  setFilter: jest.fn(),
+  a11yNotify: jest.fn()
 };
 
 beforeEach(() => {
-  params.view.mockClear();
-  params.addFilter.mockClear();
-  params.removeFilter.mockClear();
-  params.setFilter.mockClear();
```

Comment:
This is a bit of an opinionated change - I figured `clearAllMocks` was a bit more succinct than listing out each function, but if we're explicitly specifying functions for a reason, let me know and I can change this back

Reply:
Nice! I didn't know that existed. I actually have shot myself in the foot before by creating a mock and forgetting to reset it. Great call.